### PR TITLE
snapshots

### PR DESCRIPTION
**Pull request by joethorley**

### CODE REVIEW

**Review by krlmlr — APPROVED** [review]
Nice!

File: tests/testthat/test-obs-only.R [hunk 1]
```diff
@@ -18,36 +21,22 @@ test_that("new_data obs_only", {
...
-    obs_only = TRUE
-  )), 20L)
-  expect_identical(nrow(new_data(data, c("Fac1", "Fac2", "Fac3", "Random"),
-    obs_only = list(
-      "Fac1", "Fac2", "Fac3",
-      "Random"
-    )
-  )), 24L)
+
+  testthat::expect_snapshot({
```
Comment (aylapear) [inline tests/testthat/test-obs-only.R:25]: Call to `testthat` is inconsistent to the rest of your code. See above as well

**Review by aylapear — APPROVED** [review]

File: tests/testthat/test-obs-only.R [hunk 2]
```diff
@@ -6,8 +6,11 @@ test_that("obs_only", {
     circumference = c(6, 7, 8, 9, 10)
   )
   data <- Orange[c(1, 35), ]
-  expect_identical(nrow(obs_only(newdata, data, "Tree")), 50L)
-  expect_identical(nrow(obs_only(newdata, data[1, ], "Tree")), 25L)
+
+  testthat::expect_snapshot({
```
Comment (aylapear) [inline tests/testthat/test-obs-only.R]:
```suggestion
  expect_snapshot({
```

File: tests/testthat/test-obs-only.R (hunk 1 above)
Comment (aylapear) [inline tests/testthat/test-obs-only.R]:
```suggestion
  expect_snapshot({
```

**Review by aylapear — APPROVED** [review]

**Pull request merged**